Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -4154,6 +4155,36 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
     }
 }
 
+TextStream& operator<<(TextStream& ts, const RepaintRequirements& requirements)
+{
+    ts << '[';
```

Comment:
TextStream knows how to dump OptionSets. You just need to define `TextStream& operator<<` for `RepaintRequirement`